src/pretix/base/services/mail.py

```diff
@@ -52,10 +52,11 @@ class SendMailException(Exception):
     pass
 
 
-def mail(email: str, subject: str, template: Union[str, LazyI18nString],
-         context: Dict[str, Any]=None, event: Event=None, locale: str=None,
-         order: Order=None, position: OrderPosition=None, headers: dict=None, sender: str=None,
-         invoices: list=None, attach_tickets=False, auto_email=True, user=None, attach_ical=False):
+def mail(email: Union[str, Sequence[str]], subject: str, template: Union[str, LazyI18nString],
+         context: Dict[str, Any] = None, event: Event = None, locale: str = None,
+         order: Order = None, position: OrderPosition = None, headers: dict = None, sender: str = None,
+         invoices: Sequence = None, attach_tickets=False, auto_email=True, user=None, attach_ical=False,
+         attach_cached_files: Sequence = None):
```

Comment:
Downside here is that these are *a lot* of kwargs. But I don't think I can change the logic for tickets/invoices at this place.

Would be more clean to have a general-purpose `mail` and a `mail_order` that can do all the other stuff, but to delegate, invoices would need to be copied to a cached file, which we probably don't want for this often-called task.